include/mbedtls/x509_crt.h

```diff
@@ -166,6 +166,14 @@ typedef struct
 {
     mbedtls_x509_crt_verify_chain_item items[MBEDTLS_X509_MAX_VERIFY_CHAIN_SIZE];
     unsigned len;
+
+#if defined(MBEDTLS_X509_TRUSTED_CERTIFICATE_CALLBACK)
+    /* This stores the list of potential trusted signers obtained from
+     * the CA callback used for the CRT verification, if configured.
+     * We must track it somewhere because the callback passes its
+     * ownership to the caller. */
+    mbedtls_x509_crt *trust_ca_cb_result;
```

Comment:
I agree with Jarno's comment, but I don't think this needs to be changed right now, as this would either make newer code less consistent with existing names, or require changing existing names, which is what I would prefer, but probably out of scope for this PR.